### DISCUSSION TITLE
Build is queued for the past 10 minutes without any progress

### DISCUSSION THREAD

**Discussion by abarone-btf** [codemagic-ci-cd/codemagic-docs#2428] [Q&A] [answered]
### If you are looking for help, please confirm the following...

- [X] I confirm I have searched the [Docs](https://docs.codemagic.io), [Codemagic Sample Projects](https://github.com/codemagic-ci-cd/codemagic-sample-projects), and GitHub [Discussions](https://github.com/orgs/codemagic-ci-cd/discussions).
- [X] I confirm the software versions on my local computer match the ones in CI environment (e.g. Xcode version, Gradle version). See [here](https://docs.codemagic.io/troubleshooting/common-android-issues/#builds-work-locally-but-fail-on-codemagic)
- [X] I confirm I am able to do a fresh clone of the repository and build the project on my local computer.

### Which mobile framework are you using?

Flutter (Dart)

### Steps to reproduce

My builds were executing without problems yesterday, but today they are enqueued without any progress (even if I cancel them and start them again). Maybe its just slow but I know this issue has happened some times in the past and I want bring it as soon as possible.

Thanks!

### Expected results

Builds has to start in a reasonable time period

### Actual results

Enqueued forever (at least +10min)

**Comment by smartbart1** [comment]
same

**Comment by helinanever** [comment, marked as answer]
We had an incident affecting build machine availability across all machine types, please see https://status.codemagic.io/. This has been resolved now but there is a longer build queue due to that. We expect all builds to start soon. Thank you for your patience!

**Reply by gsi-eric** [reply]
The problem persists, could you tell me if there is an estimated solution time?

**Reply by helinanever** [reply]
Sadly, there has been another occurrence of the same issue -- this has been resolved now but builds may stay queued longer than usual. Our team is actively working on resolving the issue for good. We apologize for any inconvenience caused. Please check the status page for the latest info.

**Comment by himesh-cm** [comment]
Services are back to normal.
While I’m closing our discussion for now, if you have any other questions or suggestions, feel free to reply back or start a new discussion anytime. Have a nice day!
